tests/helpers.py

```diff
@@ -95,7 +95,7 @@ def wait_for_resource_state(resource, state, timeout=60):
 
 def create_pod(
     interface_type=None, pvc_name=None,
-    do_reload=True, namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+    do_reload=True, namespace=defaults.ROOK_CLUSTER_NAMESPACE, pod_name=None,
```

Comment:
Put it as last one parameter as you are adding the new one.
You are missing it in docstring.

Reply:
Done